johnwayodi/store-manager
# #161163057 Add Products View for Admin

#### What does this PR do?
Adds a view where the administrator can view all products and also create a new product
#### Description of Task to be completed?
Ensure the create new product button directs the admin to view where a new product can be added
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, store-manager by default
3. `git checkout ft-all-products-admin-161163057`
4. open products-admin.html on browser
#### Any background context you want to provide?
This page is only accessible by the administrator
#### What are the relevant pivotal tracker stories?
#161163057
#### Screenshots (if appropriate)
![admin-products](https://user-images.githubusercontent.com/10683903/46831850-3199cf00-cdad-11e8-8971-d2de09426ac6.png)

#### Questions:
Any feedback on what I can improve or change? Drop a comment or suggestion below
